src/uu/stty/Cargo.toml

```diff
@@ -16,6 +16,7 @@ path = "src/stty.rs"
 
 [dependencies]
 clap = { workspace=true }
+libc = { workspace=true }
```

Comment:
Can we keep using `nix` for this? `nix` also reexports `libc`.

Reply:
Sure, I was just looking at other utilities such as dd (I think, on phone right now, can't check) that used libc directly.

Reply:
Yes that's true. It's just that I want to move to `nix` (but without any rush) and don't want to introduce more `libc`.